### PR TITLE
DASH-1213 Show results for votes

### PR DESCRIPTION
**Pull request by trukhilio**

### CODE REVIEW

File: client/src/pages/GovernancePage/components/CastBlockProducerVotePage/components/CandidateItem/CandidateItem.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  onBlockProducerSelectChange?: (id: string) => void;
+};
+
+export const CandidateItem: React.FC<Props> = props => {
+  const {
+    hideCloseButton,
+    selectedBlockProducers,
+    onBlockProducerSelectChange,
+  } = props;
+  return (
```
Comment (andreyvEze) [inline client/src/pages/GovernancePage/components/CastBlockProducerVotePage/components/CandidateItem/CandidateItem.tsx:21]: could be changed to 
```
export const CandidateItem: React.FC<Props> =({
    hideCloseButton,
    selectedBlockProducers,
    onBlockProducerSelectChange,
  }) => (
```

File: client/src/pages/GovernancePage/components/CastBoardVotePage/components/CandidateBoardItems/CandidateBoardItems.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  onCandidateSelectChange?: (id: string) => void;
+};
+
+export const CandidateBoardItems: React.FC<Props> = props => {
+  const {
+    hideCloseButton,
+    selectedCandidates,
+    onCandidateSelectChange,
+  } = props;
+  return (
```
Comment (andreyvEze) [inline client/src/pages/GovernancePage/components/CastBoardVotePage/components/CandidateBoardItems/CandidateBoardItems.tsx:23]: ({ ... }) => (

File: client/src/pages/GovernancePage/components/ProxiesVotePage/ProxiesVotePageContext.tsx [hunk 3]
```diff
@@ -69,6 +72,7 @@ export const useContext = (props: Props): UseContextProps => {
 
   const [processing, setProcessing] = useState<boolean>(false);
   const [submitData, setSubmitData] = useState<SubmitData | null>(null);
+  const [resultsData, setResulstData] = useState<TransactionDetailsProps>(null);
```
Comment (andreyvEze) [inline client/src/pages/GovernancePage/components/ProxiesVotePage/ProxiesVotePageContext.tsx:75]:
```
useState<TransactionDetailsProps | null>
```
?

File: client/src/pages/GovernancePage/components/ProxiesVotePage/ProxiesVotePageContext.tsx [hunk 4]
```diff
@@ -127,13 +131,69 @@ export const useContext = (props: Props): UseContextProps => {
...
+              nativeFio: selectedFioWallet?.available,
+              fio: FIOSDK.SUFToAmount(selectedFioWallet?.available).toFixed(2),
+              usdc: api.fio
+                .convertFioToUsdc(selectedFioWallet?.available, roe)
+                ?.toString(),
+            },
+          },
+          additional: [
+            {
+              label: 'ID',
```
Comment (andreyvEze) [inline client/src/pages/GovernancePage/components/ProxiesVotePage/ProxiesVotePageContext.tsx:150]: should it be const somewhere?

File: client/src/pages/GovernancePage/components/ProxiesVotePage/ProxiesVotePageContext.tsx [hunk 5]
```diff
@@ -127,13 +131,69 @@ export const useContext = (props: Props): UseContextProps => {
...
+              ?.toString(),
+          };
+        } else {
+          resultsDataObj.bundles = {
+            remaining: new MathOp(selectedFioHandle?.remaining)
+              .sub(BUNDLES_TX_COUNT.VOTE_PROXY)
+              .toNumber(),
+            fee: BUNDLES_TX_COUNT.VOTE_PROXY,
+          };
+        }
```
Comment (andreyvEze) [inline client/src/pages/GovernancePage/components/ProxiesVotePage/ProxiesVotePageContext.tsx:177]: also, should we move similar calculations and object defining in some util for those components?

**Review by andreyvEze — APPROVED** [review]

File: client/src/pages/GovernancePage/components/ProxiesVotePage/ProxiesVotePageContext.tsx (hunk 4 above)
Comment (trukhilio) [reply client/src/pages/GovernancePage/components/ProxiesVotePage/ProxiesVotePageContext.tsx:150]: Done

File: client/src/pages/GovernancePage/components/ProxiesVotePage/ProxiesVotePageContext.tsx (hunk 3 above)
Comment (trukhilio) [reply client/src/pages/GovernancePage/components/ProxiesVotePage/ProxiesVotePageContext.tsx:75]: Done

File: client/src/pages/GovernancePage/components/CastBoardVotePage/components/CandidateBoardItems/CandidateBoardItems.tsx (hunk 2 above)
Comment (trukhilio) [reply client/src/pages/GovernancePage/components/CastBoardVotePage/components/CandidateBoardItems/CandidateBoardItems.tsx:23]: Done

File: client/src/pages/GovernancePage/components/CastBlockProducerVotePage/components/CandidateItem/CandidateItem.tsx (hunk 1 above)
Comment (trukhilio) [reply client/src/pages/GovernancePage/components/CastBlockProducerVotePage/components/CandidateItem/CandidateItem.tsx:21]: Done

**Review by andreyvEze — APPROVED** [review]

**Review by andreyvEze — APPROVED** [review]

**Pull request merged**